### PR TITLE
Include command to add clients [DRAFT]

### PR DESCRIPTION
**Pull request by Aishwarya-Hariharan-Iyer**
Please do not merge as this is a draft PR and the CI checks are yet to be addressed

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ClientCliSyntax.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package seedu.address.logic.parser;
+
+public class ClientCliSyntax {
+    /* Prefix definitions */
+    public static final Prefix PREFIX_CLIENT_NAME = new Prefix("n/");
+    public static final Prefix PREFIX_CLIENT_PHONE = new Prefix("p/");
+    public static final Prefix PREFIX_CLIENT_EMAIL = new Prefix("e/");
+    public static final Prefix PREFIX_CLIENT_ID = new Prefix("i/");
```
Comment (ConradLew) [inline src/main/java/seedu/address/logic/parser/ClientCliSyntax.java:8]: Is this supposed to be `PREFIX_PROJECT_ID`?

File: src/main/java/seedu/address/logic/commands/client/AddClientCommand.java [hunk 2]
```diff
@@ -20,51 +19,48 @@ public class AddClientCommand extends ClientCommand {
...
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_CLIENT_NAME + "John Doe "
+            + PREFIX_CLIENT_PHONE + "98765432 "
+            + PREFIX_CLIENT_EMAIL + "johnd@example.com "
+            + PREFIX_CLIENT_ID + "1";
 
     public static final String MESSAGE_SUCCESS = "New client added: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This client already exists in the address book";
```
Comment (ConradLew) [inline src/main/java/seedu/address/logic/commands/client/AddClientCommand.java:35]: I missed out the naming in the previous PR - kindly assist to change PERSON to CLIENT

File: src/main/java/seedu/address/logic/parser/ClientCommandParser.java [hunk 3]
```diff
@@ -46,12 +60,44 @@ public ClientCommand parse(String flag, String arguments) throws ParseException
...
      * @throws ParseException if the user input does not conform the expected format
      */
-    private AddClientCommand parseAddClientCommand(String args) throws ParseException {
-        return null;
+    private AddClientCommand parseAddClientCommand(String arguments) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(arguments, PREFIX_CLIENT_NAME, PREFIX_CLIENT_PHONE,
+                        PREFIX_CLIENT_EMAIL);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_CLIENT_NAME)
```
Comment (ConradLew) [inline src/main/java/seedu/address/logic/parser/ClientCommandParser.java:72]: Project ID should probably be compulsory as well, refer to documentation to confirm

Comment (Aishwarya-Hariharan-Iyer) [reply src/main/java/seedu/address/logic/parser/ClientCommandParser.java:71]: Done!

**Comment by Aishwarya-Hariharan-Iyer** [comment]
New PR issued with other client commands.

**Pull request closed without merging**